docs/subsystems/email.md

```diff
@@ -51,7 +51,7 @@ All the emails sent in the development environment can be accessed by
 visiting `/emails` in the browser.  The way that this works is that
 we've set the email backend (aka what happens when you call the email
 `.send()` method in Django) in the development environment to be our
-our custom backend, `EmailLogBackEnd`.  It does the following:
```

Comment:
The only change here that I'm happy about.

I'm doing a small self-review of this below, marking things that should be better.